tests/Feature/FlightControllerTest.php

```diff
@@ -2,547 +2,570 @@
 
 namespace Tests\Feature;
 
-use App\Http\Controllers\FlightController;
+use App\Actions\Flight\UpdateFlightAction;
+use App\Http\Controllers\Flight\DeleteFlightController;
+use App\Http\Controllers\Flight\GetFlightController;
+use App\Http\Controllers\Flight\StoreFlightController;
+use App\Http\Controllers\Flight\UpdateFlightController;
 use App\Models\Airline;
 use App\Models\City;
 use App\Models\Flight;
 use Illuminate\Foundation\Testing\RefreshDatabase;
 use Tests\TestCase;
 
-class FlightControllerTest extends TestCase
-{
-    use RefreshDatabase;
-
-    /** @test */
-    public function testGetFlightsForAnAirline()
-    {
-        City::factory(2)->create();
-
-        Airline::factory()->create();
-
-        Flight::create([
-            'airline_id' => 1,
-            'origin_city_id' => 1,
-            'destination_city_id' => 2,
+it('gets flights for an airline ', function () {
```

Comment:
make sure to add `uses(RefreshDatabase::class);` before the first test, so that it whipes out everything before each test!